pkg/syslog-ng-ctl/stats_remove_orphans.go

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2025 Axoflow
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package syslogngctl
+
+import "context"
+
+func StatsRemoveOrphans(ctx context.Context, cc ControlChannel) error {
+	_, err := cc.SendCommand(ctx, "REMOVE_ORPHANED_STATS")
```

Comment:
I think we should look into the response a little bit, just to be sure it is not a failure.

In case the command succeed, the response will start with `OK`:
```
OK Orphaned statistics have been removed.
```